docs/source/reference/command_line.rst

```diff
@@ -0,0 +1,690 @@
+.. _reference:command-line:
+
+******************
```

Comment:
This requires an update of the corresponding commit hook that generates it

Reply:
will do!